config/sample.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
production.json is gitignored, we don't want to push keys to git by accident.
Will add a README in next PR :)